package.json

```diff
@@ -116,7 +118,8 @@
                 "order": "asc"
               }
             }
-          ]
+          ],
+          "unused-imports/no-unused-imports": "error"
```

Comment:
I think we need to tweak the rules a bit more here...

- https://github.com/sweepline/eslint-plugin-unused-imports#react - Otherwise all imports for components will be reported unused.
- https://github.com/sweepline/eslint-plugin-unused-imports#usage - we need to disable the TS unused-vars rule and configure the new plugin to handle that.

Reply:
I wasn't sure if we wanted to use the new plugin for unused vars or stick with `@typescript-eslint/no-unused-vars`, that's why I only configured imports here. I'll updated.

Reply:
Updated to use the new plugin for unused vars. Note: The react rules you linked are already included as part of [react/recommended](https://github.com/yannickcr/eslint-plugin-react#:~:text=%E2%9C%94-,react/jsx%2Duses%2Dreact,-Prevent%20React%20to).

Reply:
Ah good to know about the react rules.

Regarding the no`@typescript-eslint/no-unused-vars` rule. If you do not disable it then one ends up getting multiple warning/error messages in eslint, which is a bit annoying.